nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# Add payment type bn

Fixes:
- Created user profile view as well as option in navbar.
- Profile page lists all payment types for user
- User can add new payment type

Changes proposed in this request & reasons behind organizational or architectural decisions:
-
-

Steps to test:
- Log in
- Click on 'Profile' in nav bar
- Click to add a new payment type
- Fill it out and add it

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
-

Commit message:
- added user profile page and hyperlink for creating a new payment type
--




Link to feature ticket:
- https://github.com/nss-day-cohort-30/bangazon-site-mighty-mahi-mahi/issues/6

@mighty-mahi-mahi
